Quick_Deploy/vLLM/model_repository/vllm/vllm_engine_args.json

```diff
@@ -1,4 +1,5 @@
 {
     "model":"facebook/opt-125m",
-    "disable_log_requests": "true"
+    "disable_log_requests": "true",
+    "gpu_memory_utilization": 0.5
```

Comment:
Add this line to the README section that's showing json contents as well

Reply:
Good catch, thanks for spotting this. Added.